viewing_party/party.py

```diff
@@ -0,0 +1,226 @@
+def create_movie(movie_title, genre, rating):
+    movie = {
+        "title": movie_title,
+        "genre": genre,
+        "rating": rating
+  
+    }
+
+    if movie_title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
```

Comment:
```suggestion
    if movie_title is None:
        return None
    if genre is None:
        return None
    if rating is None:
        return None
        
    movie = {
        "title": movie_title,
        "genre": genre,
        "rating": rating
    }

```

I'd switch the order of this function. Your guard case (the conditionals that check if any of the parameters evaluate to None) should come first before populating your movie dictionary.

Reply:
Additionally, you can collapse your conditionals using one if statement with combined expressions like so:
 
```
if not genre or not rating or not movie_title:
    return None
```